knowledge/fast_food/hungry_jacks/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: restaurant
+seed_examples: 
+- answer: Hungry Jack is founded in 19 June 1971! |
+  question: when are Hungry Jack Founded?!
+- answer:  net income for hungry Jack is 246.5 Millioin |
+  question:  what is net income for hungry Jack
+- answer: owner for Hungry Jack is Jack Cowin |
+  question:  who is owner for Hungry Jack?!
+- answer: Hungry Jack is Fastfood Resturant|
+  question: what kind if resturant is Hungry Jack 
+- answer: what is sub licence for hungrz jack ?!|
+  question: Burger king
+task_description: 'Hungry Jack's Pty Ltd. is an Australian fast food franchise of the Burger King Corporation. It is a wholly owned subsidiary of Competitive Foods Australia (with licensing from Restaurant Brands International), a privately held company owned by Jack Cowin.'
```

Comment:
You may want to use a 

```yaml
task_description: |
   Hungry Jack's Pty Ltd...
```

form for this long string since the string includes a quote char. It is also a long line and should be split up.